Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -145,6 +145,26 @@ ErrorOr<void> JPEGLoadingContext::decode()
                 }
             }
         }
+
+        // From libjpeg-turbo's libjpeg.txt:
+        // https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/libjpeg.txt
+        // CAUTION: it appears that Adobe Photoshop writes inverted data in CMYK JPEG
+        // files: 0 represents 100% ink coverage, rather than 0% ink as you'd expect.
+        // This is arguably a bug in Photoshop, but if you need to work with Photoshop
+        // CMYK files, you will have to deal with it in your application.
+        if (cinfo.saw_Adobe_marker) {
+            for (int i = 0; i < cmyk_bitmap->size().height(); ++i) {
+                for (int j = 0; j < cmyk_bitmap->size().width(); ++j) {
+                    auto const& cmyk = cmyk_bitmap->scanline(i)[j];
```

Comment:
Please call `scanline(i)` once in the outer loop and reuse that in the inner loop.